extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -115,6 +116,12 @@ class AmpCarousel extends AMP.BaseElement {
 
     /** @private {?ChildLayoutManager} */
     this.childLayoutManager_ = null;
+
+    /**
+     * Whether to show control buttons
+     * @private {boolean}
+     */
+    this.alwaysShowControls_ = false;
```

Comment:
suggestion: We can name this `showControls` instead of `alwaysShowControls`, since it can change

Reply:
done